src/components/githubRepoCard/sizeConversion.js

```diff
@@ -0,0 +1,6 @@
+export const format = value => {
```

Comment:
You could move this function into a `utils.js` file in `/src`. That way we can add more functions in one place or share it amongst files if needed

Reply:
Okay will make necessary changes.